pkg/virt-launcher/virtwrap/network/BUILD.bazel

```diff
@@ -20,10 +20,12 @@ go_library(
         "//pkg/virt-launcher/virtwrap/converter:go_default_library",
         "//pkg/virt-launcher/virtwrap/network/dhcp:go_default_library",
         "//pkg/virt-launcher/virtwrap/network/dhcpv6:go_default_library",
+        "//pkg/virt-launcher/virtwrap/network/ndp:go_default_library",
```

Comment:
Please update the commit message as the RA daemon no longer running in a new process. Since it is no longer a process, I'm not sure the name daemon is appropriate.

Reply:
Done.